src/app/routes/enrolment/enrolment.component.ts

```diff
@@ -36,7 +38,8 @@ export class EnrolmentComponent implements AfterViewInit {
   };
 
   public isMyEnrolmentShown = false;
-  isExperimentalEnabled$ = this.store.select(SettingsSelectors.isExperimentalEnabled);
+  isCreateVCVisible$ = combineLatest([this.store.select(SettingsSelectors.isExperimentalEnabled), this.store.select(RoleSelectors.hasRoles)])
+    .pipe(map(([isExperimental, hasRoles]) => isExperimental && hasRoles));
```

Comment:
I think this is either wrong or the name can be improved. Does `hasRoles` mean that the DID controls some role definitions? Or does it mean that the DID is an issuer of some roles?

Reply:
hasRole selectors returns true/false depending if the user have some roles inside role management tab inside governance. Probably i should update name for example `issuerHasRoles`

Reply:
Ok thanks. I think this behaviour is incorrect then. I don't think is relevant whether or not the DID controls roles, the relevant consideration is whether or not the DID can issue any roles.

Regarding the variable name, I think a better name would be `controlsRoles`

Reply:
Ok, got it. Now i know that i used wrong method. I think i should use `getAllowedRolesByIssuer` from domains service. I will adjust it in a moment. Thanks!